cristin-project/src/test/java/no/unit/nva/cristin/projects/query/QueryCristinProjectHandlerTest.java

```diff
@@ -214,12 +215,12 @@ void handlerReturnsNonEnrichedBodyWhenEnrichingFails() throws Exception {
         var actual = Constants.OBJECT_MAPPER.readTree(gatewayResponse.getBody());
         assertEquals(expected, actual);
     }
-    
+
     @Test
     void handlerReturnsOkWhenTitleContainsAeOeAacolon() throws Exception {
         InputStream input = requestWithQueryParameters(
             Map.of(
-                JsonPropertyNames.QUERY, RANDOM_TITLE + " æØå: " + RANDOM_TITLE,
+                JsonPropertyNames.QUERY, forceUTF8(RANDOM_TITLE + " æØå: ;., " + RANDOM_TITLE),
```

Comment:
forceUtf8 will be called implicit? Maybe we dont need to force before sending to handler? as it does it automatically when processing?

Reply:
It is not decoding url, it is making sure that String is UTF8 and not UTF16

Reply:
(to UTF16)